src/unix/fs.c

```diff
@@ -595,7 +597,10 @@ static ssize_t uv__fs_sendfile(uv_fs_t* req) {
 
     return -1;
   }
-#elif defined(__FreeBSD__) || defined(__APPLE__) || defined(__DragonFly__)
+#elif defined(__APPLE__)           || \
+      defined(__FreeBSD__)         || \
+      defined(__DragonFly__)       || \
+      defined(__FreeBSD_kernel__)
```

Comment:
For consistency, can you put the `__FreeBSD__` and `__FreeBSD_kernel__` guards together?